cpp/examples/parquet/low-level-api/run-imapala-queries.sh

```diff
@@ -0,0 +1,122 @@
+source $IMPALA_HOME/bin/impala-config.sh
```

Comment:
I don't think this file should be here?

Reply:
Yes, I will be removing after completing my tests

Reply:
it is not part of the file PR